### ISSUE TITLE
redirect to outside address when searching letter string "boy" like the tv show The Boys or flamboyant

### ISSUE THREAD

**Issue by riddie9** [btdig/dhtcrawler2#35]
http://scrikerouleausfucosan.com/

redirects to that address,which seems dead, with a long string after when searching for anything with the word "boys" in it. 

edit: any search with "boy" does this, like "flamboyant"

**Comment by ghost** [comment]
its some sort of filter to block ppl from searching for CSAM

**Comment by riddie9** [comment]
Nope. can search for the words "child" or "girl" and it shows results and none are CSAM.  There's a Netflix show called "the Boys" that has nothing to do with CSAM. Pretty sure there were results for that show before. This is something new and weird and specific to only that word and maybe that show?

**Comment by ghost** [comment]
> Nope. can search for the words "child" or "girl" and it shows results and none are CSAM. There's a Netflix show called "the Boys" that has nothing to do with CSAM. Pretty sure there were results for that show before. This is something new and weird and specific to only that word and maybe that show?

i can assure you it is. the filter is extremely fucking stupid but it is 100% a csam searching filter, just tested it with other known CSAM terms, i get redirected to the same link. your show is fine sir, just find a way to skirt around the filter and you will find it

**Comment by riddie9** [comment]
How do you skirt an absolute filter and if you can skirt the filter, doesn't that make it pointless? it blocks "flamboyant" and even "chef boyardee". I do not understand the logic of it or i guess the lack of it.

**Comment by ghost** [comment]
> How do you skirt an absolute filter and if you can skirt the filter, doesn't that make it pointless? it blocks "flamboyant" and even "chef boyardee". I do not understand the logic of it or i guess the lack of it.

I have two theories. It automatically logs your IP for future submission or possible auto-submission to LE, or it just redirects you to spyware ad-scam sites for the purpose of keeping the site running. I do not care to investigate because the link I get redirected to has no DNS records, so it was probably an ad-scam site.

Also for this specific show I have no idea but when this would happen to me I just needed to search for specific keywords that would return files or titles I expected without actually using such terms. It's incredibly hard to do this with The Boys. I'm thinking of just crawling DHT myself somehow because website has gotten really fucking annoying to use.

**Comment by Koitenshin** [comment]
The Website has always been a pain in the ass to use. You can't even "Reverse Sort" from smallest files to largest (incredibly helpful now that we have x265 encoding).

**Comment by PatoFlamejanteTV** [comment]
CP/Child Porn
